views/admin/generation/basic/fields/sql/datetime.view.php

```diff
@@ -9,4 +9,4 @@
  */
 
 ?>
-`<?= $model['column_prefix'].$field['column_name'] ?>` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
\ No newline at end of file
+`<?= $model['column_prefix'].$field['column_name'] ?>` timestamp NULL,
```

Comment:
It would be better to use the explicit syntax `NULL DEFAULT NULL` instead of the implicit syntax `NULL` for readability.